.github/scripts/trymerge.py

```diff
@@ -921,6 +921,14 @@ def gen_commit_message(self, filter_ghstack: bool = False) -> str:
         msg += f"Approved by: {approved_by_urls}\n"
         return msg
 
+    def add_numbered_label(self, label_base: str) -> None:
+        labels = self.get_labels()
+        label = label_base
+        for i in range(len(labels) if labels is not None else 0):
+            if label in labels:
+                label = f"{label_base}X{i+2}"
+        gh_add_labels(self.org, self.project, self.pr_num, [label])
```

Comment:
Does this auto-create a new label if one doesn't exist?

Reply:
Yes, according to https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#add-labels-to-an-issue